Cargo.toml

```diff
@@ -45,6 +45,11 @@ sk-cbor = { path = "libraries/cbor" }
 uuid = { version = "0.8", features = ["v4"] }
 openssl = "0.10.36"
 
+# We explicitly lock the version of those transitive dependencies because their
+# Cargo.toml don't parse with the nightly compiler used by Tock. Remove this
+# whole block once CTAP is a library.
+once_cell = { version = "=1.14", optional = true } # transitive dependency of ring
```

Comment:
This was openssl in the other PR. Also not optional.

Reply:
Done